lightning/models/cookie.py

```diff
@@ -0,0 +1,38 @@
+from lightning_adapter.models import Cookie as Cookie_gen
+from ..exceptions import CookieException
+
+
+class Cookie(dict):
+    attribute_map = {
+        'name': 'name',
+        'value': 'value',
+        'path': 'path',  # noqa: E501
+        'domain': 'domain',  # noqa: E501
+        'secure_only': 'secure',  # noqa: E501
+        'http_only': 'httpOnly',  # noqa: E501
+        'same_site_policy': 'sameSite',  # noqa: E501
+        'expires': 'expiry'}
+    swagger_types = Cookie_gen.swagger_types
+
+    def __init__(self, name, value, *args, **kwargs):
+        kwargs["name"] = name
+        kwargs["value"] = value
+
+        # adding positional arguments to kwargs if user chooses a dict type of settings
+        if args:
```

Comment:
Need a separate method that returns `Cookie_gen` instance using `Cookie` object fields.

Reply:
tried to realize it in 8c543aa7094c15a53e877f4037bc152f2c1099b3